ci-operator/step-registry/openshift/e2e/azure/private/openshift-e2e-azure-private-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+workflow:
+  as: openshift-e2e-azure-private
+  steps:
+    pre:
+    - ref: azure-provision-resourcegroup
+    - ref: azure-provision-vnet
+    - chain: azure-provision-bastionhost
+    - ref: proxy-config-generate
+    - chain: ipi-conf-azure
+    - ref: ipi-conf-azure-provisionednetwork
+    - chain: ipi-install
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: gather-installer-boostrap-logs-in-bastion
+    - chain: gather-network
+    - chain: gather-core-dump
+    - ref: gather-azure-cli
+    - ref: gather-azure-resource
+    - chain: ipi-deprovision
+    - ref: azure-deprovision-resourcegroup
+    env:
+      PUBLISH: Internal
```

Comment:
From catching issues perspective, do you have the preference to run fully private or a common private cluster ?

Reply:
I prefer to common private cluster, because in this scenario, public lb with outbound rule is still required to provide outbound traffic, while there is no public lb to be created for fully private cluster.